### ISSUE TITLE
Add D-Bus to Windows build for MPRIS support

### ISSUE THREAD

**Issue by auouymous** [gpodder/gpodder#1125]
The windows build currently uses a fake dbus module but dbus now works on windows, see https://www.freedesktop.org/wiki/Software/dbus/#windowsport.

Windows support can then be enabled in mpris-listener.py with `__only_for__ = 'freedesktop, win32'`.

**Comment by elelay** [comment]
You'll want to check that at least VLC will use dbus on windows before doing this work ;-)

**Comment by Marrib** [comment]
I think VLC does:
- https://wiki.videolan.org/DBus
- https://wiki.videolan.org/DBus-spec
- https://wiki.videolan.org/DBus-usage
- https://github.com/videolan/vlc/tree/master/modules/control/dbus

**Comment by elelay** [comment]
Yes, it does under Linux. Does any of these documents points to DBus support on windows?
